sdk/maps/azure-maps-search/azure/maps/search/_search_client.py

```diff
@@ -0,0 +1,911 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+# pylint: disable=unused-import,ungrouped-imports
+from typing import TYPE_CHECKING, overload, Union, Any, List, Optional
+from azure.core.tracing.decorator import distributed_trace
+from azure.core.credentials import AzureKeyCredential
+from azure.core.polling import LROPoller
+
+from ._base_client import MapsSearchClientBase
+from ._generated.models import (
+    PointOfInterestCategory,
+    ReverseSearchCrossStreetAddressResult,
+    SearchAlongRouteRequest,
+    GeoJsonObject,
+    BatchRequest,
+    SearchAddressBatchResult,
+    Polygon,
+)
+from .models import (
+    LatLon,
+    StructuredAddress,
+    SearchAddressResult,
+    ReverseSearchAddressResult,
+    ReverseSearchAddressBatchProcessResult,
+)
+if TYPE_CHECKING:
+    from azure.core.credentials import TokenCredential
+
+
+# By default, use the latest supported API version
+class MapsSearchClient(MapsSearchClientBase):
+    """Azure Maps Search REST APIs.
+    :param credential: Credential needed for the client to connect to Azure.
+    :type credential: ~azure.core.credentials.TokenCredential
+    :keyword api_version:
+            The API version of the service to use for requests. It defaults to the latest service version.
+            Setting to an older version may result in reduced feature compatibility.
+    :paramtype api_version: str or ~azure.ai.translation.document.MapsSearchApiVersion
+    """
+
+    def __init__(
+        self,
+        credential, # type: Union[AzureKeyCredential, TokenCredential]
+        **kwargs  # type: Any
+    ):
+        # type: (...) -> None
+
+        super().__init__(
+            credential=credential, **kwargs
+        )
+
+    @distributed_trace
+    def get_geometries(
+        self,
+        geometry_ids,  # type: List[str]
+        **kwargs  # type: Any
+    ):
+        # type: (...) -> List[Polygon]
+        """**Get Polygon**
+        `Reference Document <https://docs.microsoft.com/en-us/rest/api/maps/search/get-search-polygon>`_.
```

Comment:
Please add a proper description to the method docstring, and only link to the API reference docs if there's additional important information that cannot be summarized in the docstring.